examples/chip-tool/commands/common/DeviceScanner.cpp

```diff
@@ -17,6 +17,7 @@
  */
 
 #include "DeviceScanner.h"
+#if CHIP_DEVICE_LAYER_TARGET_DARWIN
```

Comment:
can this be a build rule thing instead to not include the cpp file for chip-tool for non-darwin?

I find that easier than a `include all CPP files and then rely on ifdefs to have code or not`

Reply:
This is already conditional in https://github.com/project-chip/connectedhomeip/blob/master/examples/chip-tool/BUILD.gn#L101

Reply:
Right, I missed this! I think I can get around that without this code change. I'll abandon this PR. Thanks!